src/components/tool_tip/index.d.ts

```diff
@@ -11,26 +13,24 @@ declare module '@elastic/eui' {
     | 'regular'
     | 'long';
 
-  export interface ToolTipProps {
+  export interface EuiToolTipProps {
+    children: ReactElement<any>;
```

Comment:
@chandlerprall So the problem I had with this one is that it's required for `EuiToolTip` and even though `EuiIconTip` has a child of `EuiIcon` TS was complaining that `EuiIconTip` required `children`.